global/config.ts

```diff
@@ -23,6 +23,7 @@ import getNextConfig from 'next/config';
 
 export const getConfig = (): Record<string, string> => {
   const publicConfig: { [k: string]: string } = getNextConfig()?.publicRuntimeConfig || {};
+  const serverConfig: { [k: string]: string } = getNextConfig()?.serverRuntimeConfig || {};
```

Comment:
What is this one being used for?

Reply:
Ahhh, I thought I deleted that. I was experimenting with some nextjs stuff and looks like I didn't clean up correctly. Removed.